charts/kubeclarity/values.yaml

```diff
@@ -41,6 +41,36 @@ kubeclarity:
 
   service:
     type: ClusterIP
+    annotations: {}
+    # Port on which Kubeclarity server is listening
+    port: 8080
+    # Target port to which the service should be mapped to
+    targetPort: 8080
+
+  ingress:
+    # Be careful when using ingress. As there is no authentication on Kubeclarity yet, Your instance may be accessible. 
+    # Make sure the ingress remains internal if you decide to enable it.
+    enabled: false
+    labels: {}
+    annotations: {}
+
+    # Optionally use ingressClassName instead of deprecated annotation.
+    # See: https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation
+    ingressClassName: ""
+
+    # As of Kubernetes 1.19, all Ingress Paths must have a pathType configured. The default value below should be sufficient in most cases.
+    # See: https://kubernetes.io/docs/concepts/services-networking/ingress/#path-types for other possible values.
+    pathType: Prefix
+
+    hosts:
+        # hostname you want to use
+      - host: chart-example.local
+        # path will default to /
+        paths: []
```

Comment:
given we only have a single backend endpoint/service, do we have a use case for exposing the same endpoint on different paths, vs just having:
```
hosts:
  - host: chart-example.local
     path: /foo
```
I was also thinking that it might be worth exposing the pathType per path too like:
```
hosts:
  - host: chart-example.local
    # defaults to /
    path: /foo
    #defaults to Prefix
    pathType: Prefix 
```
or if we do need a list:
```
hosts:
  - host: chart-example.local
     paths: 
        - path: /foo # defaults to /
          pathType: Prefix # defaults to Prefix
```

Reply:
Done, paths kept as a list with pathType configurable